### PR TITLE
Reducer specs

### PR DESCRIPTION
**Pull request by diogobeda**
Hi there, I wanted to write some tests for the app and I wanted to start by the reducers. On this first commit, I only configured the tests environment/command and wrote a sample spec so you can give your opinions on both. After we agree on a standard, I'll write the rest of the tests using them. 

What do you think?

cc @rauchg

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -26,13 +26,17 @@
...
     "babel-preset-es2015-native-modules": "6.6.0",
     "babel-preset-react": "6.11.1",
-    "eslint": "3.0.1",
+    "chai": "^3.5.0",
+    "eslint": "^3.0.1",
     "eslint-config-standard": "5.3.1",
     "eslint-plugin-promise": "1.3.2",
     "eslint-plugin-react": "5.2.2",
     "eslint-plugin-standard": "1.3.2",
+    "mocha": "^2.5.3",
```
Comment (rauchg) [inline app/package.json]: I prefer `ava` these days:
https://github.com/avajs/ava

**Comment by rauchg** [comment]
Looks awesome so far. Thank you for taking on such an important thing.

File: app/package.json (hunk 1 above)
Comment (diogobeda) [inline app/package.json]: sweet! gonna take a look at it

**Comment by diogobeda** [comment]
@rauchg I think we're all set with reducer specs. From now on, I intend to write tests to other things as well.

File: app/package.json [hunk 2]
```diff
@@ -22,17 +22,22 @@
     "seamless-immutable": "6.1.1"
   },
   "devDependencies": {
+    "ava": "^0.15.2",
+    "ava-spec": "^1.0.1",
```
Comment (rauchg) [inline app/package.json]: Can you please pin deps? Thank you!

**Comment by rauchg** [comment]
Pinging @sindresorhus in case you wanna give the ava-ness of this a look :D

File: app/package.json (hunk 2 above)
Comment (diogobeda) [inline app/package.json]: sure! will do :)

**Comment by sindresorhus** [comment]
Generally looks good, although I'm personally not a big fan of all the `describe`/`it` nesting.

-

> app/spec/reducers/sessions.spec.js

Feels moot adding `.spec` when the files are already in a `spec` directory.

**Comment by rauchg** [comment]
@diogobeda there are some merge conflicts.

**Comment by diogobeda** [comment]
Sorry, I've been very busy these days. Only gonna be able to look into it on the weekend.

**Comment by diogobeda** [comment]
Oh, the irony! I didn't have time to finish this cause I'm working on a talk I'm giving this month about testing redux. Sorry to leave it like this, guys.
@ekmartin I don't think I can dedicate a good amount of time to this, indeed. You can take it from here, if you want. Is there something I could do to make it smoother for you?

**Comment by matheuss** [comment]
Don't worry @diogobeda!! Whenever you, @ekmartin or someone else have time to work on this, a reopen/new PR is more than welcome 😄

**Pull request closed without merging**